components/scream/src/physics/p3/atmosphere_microphysics.hpp

```diff
@@ -79,13 +79,21 @@ class P3Microphysics : public AtmosphereProcess
         // Exner
         const Spack opmid(pmid(icol,ipack));
         const Smask opmid_mask(!isnan(opmid) and opmid>0.0);
-        auto oexner = physics_fun::get_exner(opmid,opmid_mask);
+        auto oexner = PF::exner_function(opmid);
         inv_exner(icol,ipack).set(opmid_mask,1.0/oexner);
         // Potential temperature
         const Spack oT_atm(T_atm(icol,ipack));
         const Smask oT_atm_mask(!isnan(oT_atm) and oT_atm>0.0);
```

Comment:
Isn't the T>0 check a bit superfluous? If T<=0 (assuming it's even possible), I believe the code should have already crashed somewhere else...